packages/app/obojobo-document-engine/src/scripts/viewer/components/viewer-app.js

```diff
@@ -412,11 +415,48 @@ export default class ViewerApp extends React.Component {
 		if (!el) {
 			return
 		}
-		if (!DOMUtil.isElementVisible(el)) {
-			return FocusUtil.clearFadeEffect()
+
+		this.observer = new IntersectionObserver(this.onIntersectionChange, {
+			root: null,
+			rootMargin: '0px',
+			threshhold: 0
+		})
+
+		this.observer.observe(el)
+	}
+
+	onIntersectionChange(changes) {
+		const change = changes[0]
+
+		if (change.intersectionRatio > 0) {
+			return false
 		}
+
+		FocusUtil.clearFadeEffect()
```

Comment:
For now we're only using the onScroll (and now, IntersectionObservers instead) to determine when to call this clearFadeEffect method, and once it's cleared it doesn't come back. So go ahead and kill the observer with `stopObservering...`